README.md

```diff
@@ -95,7 +95,10 @@ to run ingress-node-firewall-operator unit tests (no cluster required), execute
 ```shell
 make test
 ```
-
+## Generating operator bundle
```

Comment:
nit - add line:
`In order to generate an operator bundle, run the following command`: